utils/cosmos/iscn/sign.ts

```diff
@@ -47,18 +48,26 @@ export function formatISCNTxPayload(payload: ISCNRegisterPayload): ISCNSignPaylo
         ? `https://like.co/${likerIds[i]}`
         : authorUrls[i][0] || authorName
 
-      const identifiers = authorWallets[i].map((a: any) => ({
+      let identifiers = authorWallets[i].map((a: any) => ({
           '@type': 'PropertyValue',
           propertyID: WALLET_TYPE_REPLACER[a.type] || a.type,
           value: a.address,
         }))
 
+      if ( !identifiers ) {
+        identifiers = likerIds.map((a: any) => ({
```

Comment:
`likerId` should be added to `identifiers` if it exists and is valid, no matter if previously `identifiers` exists or not